docs/src/pages/guides/interoperability/StyledComponentsTheme.js

```diff
@@ -1,47 +1,36 @@
 import * as React from 'react';
-import styled, { ThemeProvider } from 'styled-components';
-import NoSsr from '@material-ui/core/NoSsr';
 import {
   createMuiTheme,
+  experimentalStyled as styled,
   ThemeProvider as MuiThemeProvider,
   darken,
 } from '@material-ui/core/styles';
-import Button from '@material-ui/core/Button';
+import Slider from '@material-ui/lab/SliderStyled';
+import Box from '@material-ui/core/Box';
 
 const customTheme = createMuiTheme({
   palette: {
     primary: {
-      main: '#6772e5',
+      main: '#20b2aa',
     },
   },
 });
 
-const StyledButton = styled(Button)`
+const CustomizedSlider = styled(Slider)`
   ${({ theme }) => `
-  background-color: ${theme.palette.primary.main};
-  color: #fff;
-  box-shadow: 0 4px 6px rgba(50, 50, 93, 0.11), 0 1px 3px rgba(0, 0, 0, 0.08);
-  padding: 4px 10px;
-  font-size: 13px;
-  &:hover {
-    background-color: ${darken(theme.palette.primary.main, 0.2)};
-  }
-  ${theme.breakpoints.up('sm')} {
-    font-size: 14px;
-    padding: 7px 14px;
+  color: ${theme.palette.primary.main};
+  :hover {
+    color: ${darken(theme.palette.primary.main, 0.2)};
   }
   `}
 `;
 
 export default function StyledComponentsTheme() {
   return (
-    <NoSsr>
-      <MuiThemeProvider theme={customTheme}>
-        <ThemeProvider theme={customTheme}>
-          <Button>Default</Button>
-          <StyledButton>Customized</StyledButton>
-        </ThemeProvider>
-      </MuiThemeProvider>
-    </NoSsr>
+    <MuiThemeProvider theme={customTheme}>
+      <Box width={300}>
```

Comment:
So we don't see the box in the preview?

```suggestion
    <Box width={300}>
      <ThemeProvider theme={customTheme}>
```